apps/assistants/models.py

```diff
@@ -85,3 +85,17 @@ def label(self):
 
     def __str__(self):
         return f"Tool Resources for {self.assistant.name}: {self.tool_type}"
+
+
+class ThreadToolResources(BaseModel):
```

Comment:
I know that this is specific to assistants currently, but we'll likely expand it's usage later so I think this should go in the `chat` app.

Reply:
https://github.com/dimagi/open-chat-studio/pull/525/commits/12f85cf7797dc7455ba67d91c4ee40719fedf264